handbook/product-design/product-design.rituals.yml

```diff
@@ -50,6 +50,20 @@
   description: "Contributors present wireframes (UI changes) that are “Ready for review”. Head of Product Design provides feedback on UI/CLI/API changes." 
   moreInfoUrl: "https://docs.google.com/document/d/1_lIlhaIpR7yYZz-sp8iE5OIipDgrq7AdZTkeoz3Mf9c/edit#"
   dri: "rachaelshaw"
+-
+  task: "🦢🔄 Product design sync"
+  startedOn: "2023-07-11" 
+  frequency: "Weekly" 
+  description: "Weekly time to chat about product design work (design reviews, conventions & best practices, using Figma, etc.)" 
+  moreInfoUrl: "https://docs.google.com/document/d/1GDEcXuTUjHI2CD9Jqega_GyF9DL6-PBmcyJpj55Lmos/edit"
```

Comment:
@noahtalerman Are the rituals confidential? If so, Ok to leave Google docs for the `moreInfoUrl`. If not, can we add them as a responsibility on the product design page. 

P.S. I've been trying to leave comments to refer back to later. Leaving some here in case

Reply:
Hey @Sampfluger88 the rituals are confidential (meetings aren't open to anyone outside of Fleet *yet*).

If I'm understanding correctly, that means we're ok with leaving the Google docs as the `moreInfoUrl`.

Is that right?

Reply:
@Sampfluger88 ping! Just giving you an extra nudge in case you missed this.

Reply:
See the most recent commit for what I'm thinking.